docs/compliance/dist/system-security-plans/ato/sc-23.md

```diff
@@ -26,6 +23,6 @@ ______________________________________________________________________
 
 ## What is the solution and how is it implemented?
 
-Add control implementation description here for control sc-23
+Customer applications fully inherit this control from cloud.gov.
```

Comment:
It's unclear to me if this refers solely to TLS sessions which are managed by cloud.gov or to all sessions in general, including authenticated user sessions which are managed by us.